.github/workflows/dispatch.yaml

```diff
@@ -0,0 +1,69 @@
+name: Deploy to dev-fss
+
+on:
+  workflow_dispatch:
+    inputs:
+      gitCommit:
+        description: "Complete git commit to deploy"
```

Comment:
Du kan vurdere å endre description til `Git ref to deploy (the complete hash):`, hvis du vil; det er sånn i fastlegerest nå.